modules/squants/src/main/scala/pureconfig/module/squants.scala

```diff
@@ -0,0 +1,184 @@
+  implicit val radianceConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Radiance](Radiance.apply, _.toString)
+  implicit val spectralIntensityConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[SpectralIntensity](SpectralIntensity.apply, _.toString)
+  implicit val spectralIrradianceConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[SpectralIrradiance](SpectralIrradiance.apply, _.toString)
+  implicit val spectralPowerConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[SpectralPower](SpectralPower.apply, _.toString)
+
+  // space
+  implicit val angleConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Angle](Angle.apply, _.toString)
+  implicit val areaConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Area](Area.apply, _.toString)
+  implicit val lengthConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Length](Length.apply, _.toString)
+  implicit val solidAngleConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[SolidAngle](SolidAngle.apply, _.toString)
+  implicit val volumeConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Volume](Volume.apply, _.toString)
+
+  // thermal
+  implicit val temperatureConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Temperature](parseTemperature, _.toString)
+  implicit val thermalCapacityConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[ThermalCapacity](ThermalCapacity.apply, _.toString)
+
+  // time
+  implicit val frequencyConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Frequency](Frequency.apply, _.toString)
+  implicit val timeConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Time](Time.apply, _.toString)
+
+  // This is temporary until https://github.com/typelevel/squants/pull/183 is released (1.2.0 ???)
+  // Without it, ScalaCheck has a pretty easy time breaking the conversion
+  private[this] def parseTemperature(s: String): Try[Temperature] = {
+    val regex = "([-+]?[0-9]*\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)[ °]*(f|F|c|C|k|K|r|R)".r
+    s match {
+      case regex(value, Fahrenheit.symbol) => Success(Fahrenheit(value.toDouble))
+      case regex(value, "f") => Success(Fahrenheit(value.toDouble))
+      case regex(value, "F") => Success(Fahrenheit(value.toDouble))
+      case regex(value, Celsius.symbol) => Success(Celsius(value.toDouble))
+      case regex(value, "c") => Success(Celsius(value.toDouble))
+      case regex(value, "C") => Success(Celsius(value.toDouble))
+      case regex(value, Kelvin.symbol) => Success(Kelvin(value.toDouble))
+      case regex(value, "k") => Success(Kelvin(value.toDouble))
+      case regex(value, "K") => Success(Kelvin(value.toDouble))
+      case regex(value, Rankine.symbol) => Success(Rankine(value.toDouble))
+      case regex(value, "r") => Success(Rankine(value.toDouble))
+      case regex(value, "R") => Success(Rankine(value.toDouble))
+      case _ => Failure(QuantityParseException("Unable to parse Temperature", s))
```

Comment:
Would you open an issue with squants for this? I'm a maintainer on squants and would be happy to look at a PR or issue.